docs/azure-config.md

```diff
@@ -142,7 +141,7 @@ velero install \
     --provider azure \
     --bucket $BLOB_CONTAINER \
     --secret-file ./credentials-velero \
-    --backup-location-config resourceGroup=<YOUR_STORAGE_RESOURCE_GROUP>,storageAccount=<YOUR_STORAGE_ACCOUNT> \
+    --backup-location-config resourceGroup=$AZURE_RESOURCE_GROUP,storageAccount=$AZURE_STORAGE_ACCOUNT_ID \
```

Comment:
needs to be $AZURE_BACKUP_RESOURCE_GROUP, which references the RG where the storage account is.  The $AZURE_RESOURCE_GROUP var captures the RG where the cluster is, which is the auto-generated one if using AKS.

Reply:
Whoops. Updated.